src/components/wb-playground/kyn-linked-div-wb/kyn-linked-div-wb.ts

```diff
@@ -0,0 +1,87 @@
+import { html, LitElement } from 'lit';
+import Styles from './kyn-linked-div-wb.scss';
+
+import '@kyndryl-design-system/shidoka-foundation/components/link';
+import { customElement, property } from 'lit/decorators.js';
+
+////// INCLUDE IN A TARGET PROP
+
+/**
+ * Linked div.
+ */
+@customElement('kyn-linked-div-wb')
+export class KynLinkedDivWb extends LitElement {
+  static override styles = [Styles];
+
+  /**
+   * Item size, required.
+   * @type {string}
+   * @default 'auto'
+   */
+  @property({ type: String })
+  size = 'auto';
+
+  /**
+   * Title text, required.
+   * @type {string}
+   * @default ''
+   */
+  @property({ type: String })
+  divTitle = '';
+
+  /**
+   * Main header text, required.
+   * @type {string}
+   * @default ''
+   */
+  @property({ type: String })
+  mainHeader = '';
+
+  /**
+   * Background hover color.
+   * @type {string}
+   * @default '#23564d'
+   */
+  @property({ type: String })
+  hoverColor = '#23564d';
+
+  /**
+   * Body text, required.
+   * @type {string}
+   * @default ''
+   */
+  @property({ type: String })
+  body = '';
+
+  /**
+   * External/internal url for kd-link, required.
+   * @type {string}
+   * @default ''
+   */
+  @property({ type: String })
+  linkHref = '';
+
+  override render() {
+    return html`
+      <div
+        id="linked-div"
+        class=${this.size}
+        style="--hover-color: ${this.hoverColor}"
+      >
+        <kd-link standalone href=${this.linkHref}>
```

Comment:
I maybe would just use an `<a>` tag in this case. The kd-link component has more specific styes and use cases.